### ISSUE TITLE
Undefined index: access_token after updating to 2.0.0

### ISSUE THREAD

**Issue by mathieuhays** [facebookincubator/Facebook-Pixel-for-Wordpress#10]
I get the following notice after updating the plugin from 1.8.0 to 2.0.0:

```
Notice: Undefined index: access_token in /srv/www/public_html/content/plugins/official-facebook-pixel/core/FacebookWordpressOptions.php on line 74
```

**Comment by davideprevosto** [comment]
I have the same here:

`18-Mar-2020 06:57:39 UTC] PHP Notice:  Undefined index: access_token in /var/www/example.com/htdocs/releases/6/wp-content/plugins/official-facebook-pixel/core/FacebookWordpressOptions.php on line 74`

**Comment by mathieuhays** [comment]
Looks like it has been fixed yesterday in https://github.com/facebookincubator/Facebook-Pixel-for-Wordpress/commit/7a17953057415cb8dbbdd02e30bfa833b3e63305. Not sure when they plan to release the fix.

**Comment by mathieuhays** [comment]
Fixed with 2.0.1 release
